src/Sulu/Bundle/ContactBundle/Resources/config/services.xml

```diff
@@ -45,7 +45,6 @@
             <argument type="service" id="sulu.repository.media" />
             <argument type="service" id="sulu_event_log.domain_event_collector" />
             <argument type="service" id="sulu.repository.user" />
-            <argument type="string">%sulu.model.account.class%</argument>
```

Comment:
😂

Reply:
It's always a pleasure to fix something like that 🙈